modules/ensemble/lib/widget/button.dart

```diff
@@ -148,6 +154,39 @@ class ButtonState extends EWidgetState<Button> {
   }
 
   Widget _buildButtonChild() {
+    if (widget._controller.loading == true) {
+      // Measure text size to keep the button consistent
+      final textSpan = TextSpan(
```

Comment:
this could be the default loading container. But we still need to provide the ability for devs to put their own custom widget as a loading container. Please see https://github.com/EnsembleUI/ensemble/issues/1584